### ISSUE TITLE
input type ="range" issue rangeslider.js

### ISSUE THREAD

**Issue by Bindhyachal** [pa11y/pa11y#378]
Hi Team,

I am using the rangeslider.js plugin for my video component on the webpage which works on input type="range" gives below error on running pa11y version 4.13.0
![pa11y-error](https://user-images.githubusercontent.com/30178954/36016686-39896fb4-0d9a-11e8-82b3-98146baf370e.JPG)

have tried using attibutes like "name,aria-label,aria-labelledBy" and also adding label element before the input but it does not work.

Kindly help me with the solution.

**Comment by rowanmanning** [comment]
This seems broken, I've opened a corresponding issue in HTML CodeSniffer, which we use for our a11y testing under the hood: https://github.com/squizlabs/HTML_CodeSniffer/issues/210

In the meantime, you could hide range elements while waiting for a fix. Add the following to the end of your Pa11y command:

```
--hide-elements "input[type=range]"
```

**Comment by rowanmanning** [comment]
My issue has been fixed in HTML CodeSniffer, we just need to wait for them to release a new version then this will be fixed.
